docs/source/error_code_list2.rst

```diff
@@ -612,3 +612,51 @@ Example:
     # mypy: disallow-any-explicit
     from typing import Any
     x: Any = 1  # Error: Explicit "Any" type annotation  [explicit-any]
+
+
+.. _code-exhaustive-match:
```

Comment:
I'm not sure if the others have duplicated docs but if not could you just link from one of these sections to the other?